cumulus_library/studies/core/core_templates/condition.sql.jinja

```diff
@@ -0,0 +1,51 @@
+{% import 'core_utils.jinja' as utils %}
+
+CREATE TABLE core__condition AS
+WITH temp_condition AS (
+    SELECT
+        concat('Condition/', c.id) AS condition_ref,
+        {{- utils.basic_nullable_cols(
```

Comment:
This should never fail and return null though yeah? Cumulus ETL guarantees all toplevel columns will be there, I would have thought.

Reply:
this is probably paranoid, but might be useful if we hit a bug or something - the implicit goal here is to make library runs less failure prone, so some 'lets verify even though we're sure' is in spirit.

Although - and I just created #164 to remember to track this - we should track cases where we hit one of these and report it back in some fashion.